src/onPreprocess/defineMeasureData.js

```diff
@@ -19,9 +18,4 @@ export default function defineMeasureData() {
 
         return !filtered;
     });
-    this.nested_measure_data = nest()
```

Comment:
Guessing this wasn't be used?

Reply:
Right, only needed that in safety-results-over-time for the box plots.